simulator/view_manager.go

```diff
@@ -249,15 +249,16 @@ func (v *ListView) update(ctx *Context) {
 	ctx.Map.Update(v, []types.PropertyChange{{Name: "view", Val: v.View}})
 }
 
-func (v *ListView) add(ctx *Context, refs []types.ManagedObjectReference) *types.ManagedObjectNotFound {
+func (v *ListView) add(ctx *Context, refs []types.ManagedObjectReference) []types.ManagedObjectReference {
+	var unresolved []types.ManagedObjectReference
 	for _, ref := range refs {
 		obj := ctx.Session.Get(ref)
 		if obj == nil {
-			return &types.ManagedObjectNotFound{Obj: ref}
```

Comment:
Does this mean that before this change, if I added 10 task MoRefs at once, and the first one happened to be expxired, `add()` would directly return a fault to the simulator side `ModifyListView()` without adding the rest 9 tasks?

Reply:
That's right. And with the change, all valid refs are added to the view and all invalid refs are returned.